ably/auth.go

```diff
@@ -70,7 +70,7 @@ type Auth struct {
 
 	// onExplicitAuthorize is the callback that Realtime sets to reauthorize with the
 	// server when Authorize is explicitly called.
-	onExplicitAuthorize func(context.Context, *TokenDetails)
+	onExplicitAuthorize func(context.Context, *TokenDetails) error
```

Comment:
Since this is related to realtime, can we should call it `authorizeRealtime`, so method call actually makes sense.
Also add spec id on top of the method